test/stat_tracker_test.rb

```diff
@@ -437,112 +449,55 @@ def test_it_gets_rival
+    assert_equal expected, @@stat_tracker.game_ids_in_season("20142015")
+  end
 
-    assert_equal "Real Salt Lake", stat_tracker.most_accurate_team("20132014")
-    assert_equal "Toronto FC", stat_tracker.most_accurate_team("20142015")
+  def test_it_gets_most_accurate_team
+    assert_equal "Real Salt Lake", @@stat_tracker.most_accurate_team("20132014")
+    assert_equal "Toronto FC", @@stat_tracker.most_accurate_team("20142015")
   end
 
   def test_it_gets_least_accurate_team
-    skip
-    game_path = './data/games.csv'
-    team_path = './data/teams.csv'
-    game_teams_path = './data/game_teams.csv'
-
-    locations = {
-      games: game_path,
-      teams: team_path,
-      game_teams: game_teams_path
-    }
-
-    stat_tracker = StatTracker.from_csv(locations)
-
-    assert_equal "New York City FC", stat_tracker.least_accurate_team("20132014")
-    assert_equal "Columbus Crew SC", stat_tracker.least_accurate_team("20142015")
+    assert_equal "New York City FC", @@stat_tracker.least_accurate_team("20132014")
+    assert_equal "Columbus Crew SC", @@stat_tracker.least_accurate_team("20142015")
   end
 
   def test_it_gets_most_tackles
     skip
-    game_path = './data/games.csv'
-    team_path = './data/teams.csv'
-    game_teams_path = './data/game_teams.csv'
-
-    locations = {
-      games: game_path,
-      teams: team_path,
-      game_teams: game_teams_path
-    }
-
-    stat_tracker = StatTracker.from_csv(locations)
-
-    assert_equal "FC Cincinnati", stat_tracker.most_tackles("20132014")
-    assert_equal "Seattle Sounders FC", stat_tracker.most_tackles("20142015")
+    assert_equal "FC Cincinnati", @@stat_tracker.most_tackles("20132014") # PASS
```

Comment:
We should look back through these as a group and see if it's our methods that are not working properly, or if the data loaded vis the csv files are different and the spec harness is drawing a different data set than what we are looking at.